include/rocksdb/pluggable_compaction.h

```diff
@@ -39,6 +39,9 @@ class PluggableCompactionParam {
 
   // The level to which the files are compacted into
   int output_level;
+
+  // Control whether the DB is shutting down
+  const std::atomic<bool>* shutting_down;
```

Comment:
Make this `shared_ptr`, it's cheap and safer.

Reply:
Oh, I guess this is from RocksDB code, in which case it's fine to leave as it.

Reply:
@igorcanadi I think this variable is passed all the way from RocksDB, so I don't think I can change it without significantly changing RocksDB. Any tips?